app/DailyMenu.php

```diff
@@ -3,8 +3,19 @@
 namespace App;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
 
 class DailyMenu extends Model
 {
-    //
+    use softDeletes;
+
+    /**
+     * Get all of foods on menu by day
```

Comment:
`Get all of foods on menu by day` this comment is not correct because this function doesn't get foods, it's just a relationship function. 
Just comment like this is ok: `DailyMenu has many Food`